tools/check-imports.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import glob
+import os
+import re
+import sys
+
+
+def do_exist(file, lines, imported):
```

Comment:
What would you think of using `filename` instead of `file`? I’d find that more a bit readable + it doesn’t shadow the `file` built-in.